lms/djangoapps/courseware/views.py

```diff
@@ -1513,7 +1513,10 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True):
             request, unicode(course_key), unicode(usage_key), disable_staff_debug_info=True, course=course
         )
 
+        responsive = block.has_support(getattr(block, 'student_view', None), 'multi_device')
```

Comment:
Should the viewport always be set for mobile web views, and always *not* set for desktop views?
If so, the responsive flag should not be determined from the block.  But rather, from the caller of this view.
The LTI endpoint may not want the viewport, while the endpoint called by mobile will want the viewport.  If so, you'll want it to be an optional parameter to this view.

Also, I want to be able to render an xBlock with the viewport set even if it hasn't been properly declared as responsive.